engine/src/main/scala/cromwell/webservice/routes/wes/WesRouteSupport.scala

```diff
@@ -228,7 +228,7 @@ object WesRouteSupport {
     val metadataJsonResponse = metadataBuilderActorRequest(workflowId, request, serviceRegistryActor)
 
     metadataJsonResponse.map {
-      case SuccessfulMetadataJsonResponse(_, responseJson) => WesResponseWorkflowMetadata(WesRunLog.fromJson(responseJson.toString()))
+      case SuccessfulMetadataJsonResponse(_, responseJson) => WesRunLog.fromJson(responseJson.toString())
```

Comment:
Do we also need to update response definition in Swagger?

Reply:
No, this works without having to update it.